### PR TITLE
Added mytool Tutorial and supporting data

### PR DESCRIPTION
**Pull request by yunyicheng**

### CODE REVIEW

File: tools/.DS_Store
Comment (ababaian) [inline tools/.DS_Store:1]: please remove this file from pull request

File: tools/img/.DS_Store
Comment (ababaian) [inline tools/img/.DS_Store:1]: ditto

File: tools/img/muscle/.DS_Store
Comment (ababaian) [inline tools/img/muscle/.DS_Store:1]: ditto

**Comment by yunyicheng** [comment]
Thanks for the advice! I have deleted these files

**Pull request merged**